ambassador/ambassador/ir/irtls.py

```diff
@@ -54,11 +54,24 @@ def setup(self, ir: 'IR', aconf: Config):
         if not self.enabled:
             return False
 
+        self['valid_tls'] = False
         secret = self.get('secret')
 
         # Check if secret and certs, both are specified
         cert_specified = False
-        if self.get('cert_chain_file') is not None or self.get('private_key_file') is not None:
+
+        cert_chain_file = self.get('cert_chain_file')
+        if cert_chain_file is not None:
+            if not ir.file_checker(cert_chain_file):
+                self.logger.error('cert_chain_file path {} does not exist'.format(cert_chain_file))
```

Comment:
These should really be calling `self.post_error()` -- that'll log the error but also record it so that the diag UI can show it.

(If the diag UI _doesn't_ show it, that's a bug and we'll fix it.)

Reply:
fixed